client/components/widget/data_viz/gviz/chart-config-directive.html

```diff
@@ -26,39 +26,112 @@
                  ng-model="ngModel.layout.columnspan">
         </div>
       </div>
-
+    </div>      
+      
+    <div class="sidebar-group">
+      <div class="sidebar-group-title">Chart</div>
+  
       <div class="sidebar-item">
         <div class="sidebar-item-label">chart type</div>
```

Comment:
I added an M23 bug to reflect the change to the sidebar UX.  Implicit in this is a CSS overhaul.